shared/actions/chat.js

```diff
@@ -179,7 +179,7 @@ function selectConversation (conversationIDKey: ConversationIDKey, fromUser: boo
   return {type: 'chat:selectConversation', payload: {conversationIDKey, fromUser}}
 }
 
-function _inboxConversationToConversation (convo: ConversationLocal, author: ?string, following: {[key: string]: boolean}, metaData: MetaData): ?InboxState {
+function _inboxConversationToConversation (convo: ConversationLocal, author: ?string, following: {[key: string]: boolean}, metaData: MetaData, youCreatedConversationIDKey: ?ConversationIDKey): ?InboxState {
```

Comment:
i had that originally but that meant the callers needed to extract the conversation id of the convo out of convo.info etc w/ all the null checks